src/itensor/oneitensor.jl

```diff
@@ -42,3 +42,39 @@ end
 function emptyITensor(elt::Type{<:Number}=NDTensors.default_eltype())
   return emptyITensor(elt, Index(0))
 end
+
+# To fix ambiguity with QN version
+function randomITensor(::Type{ElT}, is::Tuple{}) where {ElT<:Number}
+  return randomITensor(Random.default_rng(), ElT, Index(0))
```

Comment:
I don't think it is right to use `Index(0)`. Can't it just have an empty set of indices?

Reply:
In that case it seems like you are just missing an explicit definition like:
```julia
ITensor(aliasstyle::AllowAlias, elt::Type{<:Number}, A::AbstractArray, inds::Tuple{})
```
where you can define it to create a tensor with `Dense` storage.